examples/hosts/app-integration/external-controller/src/app.ts

```diff
@@ -79,6 +79,10 @@ async function start(): Promise<void> {
         ? await client.createContainer({ id: containerId, logger: consoleLogger }, containerSchema)
         : await client.getContainer({ id: containerId, logger: consoleLogger }, containerSchema);
 
+    if (createNew) {
+        client.attach(containerId);
```

Comment:
this can chance to passing in the container after #7162 is merged. Future API will be simply passing container in, but for now we need the ID to match it up.